api/src/pcapi/alembic/versions/20250218T164119_060ef534e1a5_add_future_offer_reminders.py

```diff
@@ -0,0 +1,39 @@
+"""Add future offers reminders
+
+add_future_offer_reminders
+"""
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# pre/post deployment: pre
+# revision identifiers, used by Alembic.
+revision = "060ef534e1a5"
+down_revision = "1ec374800b10"
+branch_labels: tuple[str] | None = None
+depends_on: list[str] | None = None
+
+
+def upgrade() -> None:
+    op.create_table(
+        "future_offer_reminder",
+        sa.Column("id", sa.BigInteger(), autoincrement=True, nullable=False),
+        sa.Column("user_id", sa.BigInteger(), nullable=False),
+        sa.Column("offer_id", sa.BigInteger(), nullable=False),
+        sa.ForeignKeyConstraint(
+            ["offer_id"],
+            ["offer.id"],
+        ),
+        sa.ForeignKeyConstraint(
+            ["user_id"],
+            ["user.id"],
+        ),
```

Comment:
```suggestion
        sa.ForeignKeyConstraint(
            ["offer_id"],
            ["offer.id"],
            postgresql_not_valid=True,
        ),
        sa.ForeignKeyConstraint(
            ["user_id"],
            ["user.id"],
            postgresql_not_valid=True,
        ),
```
ces deux tables sont immenses et la contrainte de FK prend un verrou sur les deux

il faut d'abord créer la contrainte en non valide et la valider dans une migration post, tu peux prendre example sur les migrations du commit 0050d8e45d46092ef4ec1534ab0d50c8625588d1